etc/nginx/nginx.conf

```diff
@@ -1,5 +1,5 @@
 user                    nginx;
-worker_processes        4;
+worker_processes        8;
```

Comment:
I was thinking a little more about this. I'm not sure we want to use the same worker processes as cores. 

We have 4 hosts w/ 8 cors, but 12 running tasks in 6 services.  The individual tasks are granted `Task CPU (unit)3072`

The [reference examples](https://github.com/awslabs/ecs-nginx-reverse-proxy/blob/master/reverse-proxy/nginx/nginx.conf) from aws leave the worker_processes at the default - 1.They set the worker_connections to 3x the CPU in teh task definition. I'm not sure how deliberate those choices are. 

I'd suggest we leave this as is so we can ship the other bits, do more research and reach out to AWS dev support for general guidance.  What do you think?

Reply:
@Rio517 I agree, going to remove the worker process increase from this pr and postpone it until we have a bit more time to benchmark and add some monitoring to see how we are utilising the resources available to us.